### ISSUE TITLE
Create a Front end for this qr code scanner

### ISSUE THREAD

**Issue by Jeevan1310** [Jeevan1310/cli-qr-scanner#1]
A live website that  can scan a qr code realtime

**Comment by beyfendipenguen** [comment]
hi. i can help you.Furthermore, i have done a similar project

**Comment by Jeevan1310** [comment]
> 

Yeah sure

**Comment by decoded-cipher** [comment]
@beyfendipenguen @Jeevan1310 
A front-end for the project will a great way for the common people to make use of these projects...

**Comment by Jeevan1310** [comment]
Sure @decoded-cipher

**Comment by beyfendipenguen** [comment]
i prefer command line interface than graphic user interface. However so many people don't agree with me :smile:

**Comment by Jeevan1310** [comment]
> i prefer command line interface than graphic user interface. However so many people don't agree with me 😄

I agree with you but the common users need graphics to do their work 🙂
